styles/LoginDropdown.module.css

```diff
@@ -62,14 +63,6 @@
   padding: 0 !important;
 }
 
-.loginOption button {
-  background: none;
-  border: none;
-  cursor: pointer;
-  border-radius: 5px;
-  font-weight: 600;
-}
-
```

Comment:
why you removed this?

Reply:
The same code was written twice.

Reply:
please revert this! I did not see any same code in file. if it is there please point. and other thing as you can se we have used `px`. can you make it responsive?
we have some -
EM: Relative to the parent element
REM: Relative to the root element (HTML tag)
%: Relative to the parent element
VW: Relative to the viewport’s width
VH: Relative to the viewport’s height

Reply:
![Screenshot (438)](https://github.com/anmode/grabtern-frontend/assets/93856491/9d51f531-939c-48f6-a5e1-8c80a3f42763)
 I don't know how can I point to show but I have added the screenshot so that u can see the exact code was written twice and I have also added font-size and changed the font weight in it.

Reply:
> please revert this! I did not see any same code in file. if it is there please point. and other thing as you can se we have used `px`. can you make it responsive? we have some - EM: Relative to the parent element REM: Relative to the root element (HTML tag) %: Relative to the parent element VW: Relative to the viewport’s width VH: Relative to the viewport’s height

Yup, I will try to make it responsive.